lib/experiments-page.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * Bootstraping the Gutenberg experiments page.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * The main entry point for the Gutenberg experiments page.
+ *
+ * @since 5.2.3
```

Comment:
Looking at other PHP files in the folder, this should be the Gutenberg version rather than the Core version.

Right now, it's not worth worrying about changing it. I think wait until the PR is approved and then update the version numbers with the upcoming Gutenberg version.